README.md

```diff
@@ -9,6 +9,10 @@ To pass the Git-101 challenge you need to get a Pull Request merged into this re
 Acceptance Criteria:
 
 1. Create a new branch in [this repo](https://github.com/wedoforce/students) called `student/<first name>-<last name>`. E.g. `student/nikita-chursin`
-1. Pull request (PR) is raised against the `main` branch
+Created branch student/anna-blumenkrantz
```

Comment:
Please revert these changes to the main branch files. You can easily do this via the following command `git checkout main README.md` and then commit changes